.github/workflows/maven.yml

```diff
@@ -6,7 +6,7 @@ name: Maven unit tests
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
We can also remove `master` from here:
https://github.com/atlassian-labs/data-center-helm-charts/blob/main/.github/workflows/documentation.yaml#L5